Dockerfile.assisted-service.ocp

```diff
@@ -19,6 +19,9 @@ FROM registry.ci.openshift.org/ocp/4.14:base
 
 LABEL io.openshift.release.operator=true
 
+# multiarch images need it till WRKLDS-222 and https://bugzilla.redhat.com/show_bug.cgi?id=2111537 are fixed
+RUN dnf install -y --setopt=install_weak_deps=False skopeo
```

Comment:
All dnf installations must occur in the same step to minimize image size (see `dnf clean all` below)

Reply:
@omertuc I think I misunderstood. Did you want me to add `dnf clean all` or add `skopeo` to the step below?

Reply:
No, I wanted all dnf actions to be in the same step